mrtt-ui/src/components/SiteBackgroundForm.js

```diff
@@ -0,0 +1,383 @@
+import { useState } from 'react'
+import axios from 'axios'
+import { useForm, useFieldArray, Controller } from 'react-hook-form'
+import { yupResolver } from '@hookform/resolvers/yup'
+import * as Yup from 'yup'
+import {
+  Box,
+  Button,
+  Checkbox,
+  Chip,
+  FormLabel,
+  List,
+  ListItem,
+  MenuItem,
+  OutlinedInput,
+  Select,
+  TextField,
+  Typography
+} from '@mui/material'
+
+import { MainFormDiv, FormQuestionDiv } from '../styles/forms'
+import {
+  areStakeholdersInvolvedOptions,
+  customaryRightsOptions,
+  governmentArrangementOptions,
+  landTenureOptions,
+  lawOptions,
+  managementStatusOptions,
+  protectionStatusOptions,
+  stakeholderOptions
+} from '../data/projectDetailsOptions'
+// TODO: update question mapping to include 2.0 questions
+import { questionMapping } from '../data/questionMapping'
+
+const ProjectDetailsForm = () => {
+  // form validation rules
+  const validationSchema = Yup.object().shape({
+    stakeholders: Yup.array().of(
+      Yup.object().shape({
+        stakeholderType: Yup.string(),
+        stackholderName: Yup.string()
+      })
+    ),
+    managementStatus: Yup.string(),
+    lawStatus: Yup.string(),
+    managementArea: Yup.string(),
+    protectionStatus: Yup.array().of(Yup.string()),
+    areStakeholdersInvolved: Yup.string(),
+    govermentArrangement: Yup.array().of(Yup.string()),
+    landTenure: Yup.array().of(Yup.string()),
+    customaryRights: Yup.string()
+  })
+  const formOptions = { resolver: yupResolver(validationSchema) }
+
+  // get functions to build form with useForm() and useFieldArray() hooks
+  const { handleSubmit, formState, control } = useForm(formOptions)
+  const { errors } = formState
+  const {
+    fields: stakeholdersFields,
+    append: stakeholdersAppend,
+    remove: stakeholdersRemove
+  } = useFieldArray({ name: 'stakeholders', control })
```

Comment:
Neat. TIL about `useFieldArray`. Also nice call on renaming. React Hook Form uses some seriously generic/vague default names!

Reply:
Also I like this:
![Screen Shot 2022-05-16 at 12 20 13 PM](https://user-images.githubusercontent.com/1740152/168657448-e5f51920-ce87-459b-883d-afea9439cc85.png)

Reply:
renaming these also allows us to have multiple fieldArrays in one form :)